apps/src/StudioApp.js

```diff
@@ -1579,8 +1588,8 @@ StudioApp.prototype.handleHideSource_ = function (options) {
       $('body').append(wireframeSendToPhone);
     }
     document.body.style.backgroundColor = '#202B34';
-  // For share page on mobile, do not show this part.
-  } else if (!options.embed && !(this.share && dom.isMobile())) {
+  // For share page, do not show this part.
+  } else if (!options.embed && !this.share) {
```

Comment:
My hypothesis is that this is dead code now, but I only say that with like 70% confidence.